evap/results/templates/results_evaluation_detail.html

```diff
@@ -28,8 +28,16 @@
         {% endif %}
         <div class="d-flex">
             <h3 class="mr-auto">{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
-
-            <div class="btn-switch btn-switch-light my-auto d-print-none">
+        </div>
+        <div class="d-flex mb-3">
```

Comment:
please place the button next to the grade download buttons in the overview card div